data/json/npcs/BG_traits.json

```diff
@@ -697,9 +697,9 @@
     "type": "mutation",
     "//": "This NPC cannot tell you about other NPCs.",
     "id": "BOSS_No_Others_01",
-    "name": { "str": "Other survivors story" },
+    "name": { "str": "Other Survivors Story" },
```

Comment:
Should these have an apostrophe somewhere on Survivors?

Reply:
I was thinking about it too; whether it should be "Other Survivor Story", "Other Survivor's Story" or "Other Survivors Story" (plural).

Reply:
I don't know the context of these, so I wasn't sure which, if any it should have, but figured it was worth mentioning

Reply:
I was one of the authors on this, these mutations determine whether or not an NPC knows about other survivors. The current formatting is correct, though capitalization is fine